packages/components/ng-log-to-customer/src/service.js

```diff
@@ -1,11 +1,24 @@
 export default class LogToCustomerService {
   /* @ngInject */
-  constructor($http) {
+  constructor($http, iceberg) {
     this.$http = $http;
+    this.iceberg = iceberg;
   }
 
-  getLogSourceUrl(source) {
-    return this.$http.post(source).then(({ data }) => data);
+  getLogsKind(url) {
```

Comment:
This function can retrieve any resource not only LogsKind, maybe the naming is not correct

Reply:
You're right, but in this case the function would simply be called "function get(url)" since it performs a simple get. I named it that way so you know what it's going on without having to open the service. Should I make the change to "function get(url)" ?

Reply:
`icebergQuery` would be more appropriate as you're using it